src/components/common/HighlightsView.vue

```diff
@@ -378,10 +374,11 @@ export default defineComponent({
     statisticsDictToChartData: function (stats) {
       let entries = Object.entries(stats)
       return {
-        labels: entries.map(e => { return e[0] }),
+        labels: entries.map((e) => { return e[0] }),
         datasets: [
           {
-            data: entries.map(e => { return e[1] }),
+            data: entries.map((e) => { return e[1] }),
+            // Color pallete for pie charts.
```

Comment:
s/pallete/palette. I guess I meant "what palette is this? where did you get it?" not "what are these colors for?"

Reply:
I'll add a comment for it. I wanted to just use some brewer palette but they topped out at 12 colors and I think we already have more than 12 reference genomes (which I'm sort of worried about as more get added). I ended up just googling for differentiated colors and got these from here: https://sashamaps.net/docs/resources/20-colors/. There is probably a better way than just a list of colors though.

Reply:
list of colors is fine as long as you put where you got it